src/components/Button.tsx

```diff
@@ -6,7 +6,9 @@ import { textSet, TextSetProps } from "onno-react";
 
 type ButtonProps = BoxProps & TextSetProps;
 
-interface Props extends ButtonProps {}
+interface Props extends ButtonProps {
+  type?: "submit" | "button" | "reset";
```

Comment:
why not add   `React.HTMLAttributes<HTMLButtonElement>` to the `type ButtonProps` up top instead? I think thats how we are handling html attrs right now